packages/synthetic-chain/src/lib/vat-status.js

```diff
@@ -92,14 +92,16 @@ const buildSwingstoreTool = () => {
 /**
  * @param {SwingstoreTool} kStore
  * @param {string} vatID
+ * @returns {{vatName: string, vatID: string, incarnation: number, terminated: boolean}}
```

Comment:
seems incidental, but: what does `...source` expand to? Or it just overrides vatName/vatID/incarnation?

Reply:
I think it has `bundleID`.